JENKINS-55834 Typo in hudsonHomeIsFull, because of escaping apostrophes

Francois-D opened the pull request:
See (https://issues.jenkins-ci.org/browse/JENKINS-55834).

<!-- Comment: 
If the issue is not fully described in the ticket, add more information here (justification, pull request links, etc.).

 * We do not require JIRA issues for minor improvements.
 * Bugfixes should have a JIRA issue (backporting process).
 * Major new features should have a JIRA issue reference.
-->

### Proposed changelog entries
Update typo from "It'll to It will"

<!-- Comment: 
The changelogs will be integrated by the core maintainers after the merge.  See the changelog examples here: https://jenkins.io/changelog/ -->

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

<!-- For new API and extension points: Link to the reference implementation in open-source (or example in Javadoc) -->

### Desired reviewers

@mention

<!-- Comment:
If you need an accelerated review process by the community (e.g., for critical bugs), mention @jenkinsci/code-reviewers
-->

Francois-D committed:
Update index.properties

oleg-nenashev reviewed (approved):
Thanks @Francois-D !

daniel-beck commented on core/src/main/resources/hudson/diagnosis/HudsonHomeDiskUsageMonitor/index.properties:
```diff
@@ -23,7 +23,7 @@
 description.1=\
   Your <tt>JENKINS_HOME</tt> ({0}) is almost full. \
-  When this directory completely fills up, it\'ll wreak havoc because Jenkins can\'t store any more data.
+  When this directory completely fills up, it will wreak havoc because Jenkins can\'t store any more data.
```
Wouldn't `can't` have the same problem on the same line?

Francois-D replied:
Thanks for pointing out this issue.  I've made the edits.

Whenever you are using MessageFormat you should be aware that the single quote character (') fulfils a special purpose inside message patterns. The single quote is used to represent a section within the message pattern that will not be formatted. A single quote itself must be escaped by using two single quotes ('').

Francois-D committed:
Update index.properties

Whenever you are using MessageFormat you should be aware that the single quote character (') fulfils a special purpose inside message patterns. The single quote is used to represent a section within the message pattern that will not be formatted. A single quote itself must be escaped by using two single quotes ('').

Francois-D committed:
Update index.properties

Changed "Can't" to can not

daniel-beck commented on core/src/main/resources/hudson/diagnosis/HudsonHomeDiskUsageMonitor/index.properties:
```diff
@@ -23,7 +23,7 @@
 description.1=\
   Your <tt>JENKINS_HOME</tt> ({0}) is almost full. \
-  When this directory completely fills up, it\'ll wreak havoc because Jenkins can\'t store any more data.
+  When this directory completely fills up, it will wreak havoc because Jenkins can not store any more data.
```
```suggestion
  When this directory completely fills up, it will wreak havoc because Jenkins cannot store any more data.
```

daniel-beck committed:
Update core/src/main/resources/hudson/diagnosis/HudsonHomeDiskUsageMonitor/index.properties

Co-Authored-By: Francois-D <46973576+Francois-D@users.noreply.github.com>

alecharp commented:
Thank you @Francois-D for your contribution! Will be merged soon-ish.
